src/NuGet/Microsoft.CodeAnalysis.EditorFeatures.nuspec

```diff
@@ -10,14 +10,19 @@
 
       Supported Platforms:
       - .NET Framework 4.6
-      
+
       More details at https://aka.ms/roslyn-packages
 
       $commitPathMessage$
     </description>
     <dependencies>
       <dependency id="Microsoft.CodeAnalysis.Features" version="[$version$]" />
```

Comment:
Let's also add `<dependency id="Microsoft.CodeAnalysis.EditorFeatures.Text" version="[$version$]" />` here

Reply:
None of the 3 listed dlls (Microsoft.CodeAnalysis.EditorFeatures.dll, Microsoft.CodeAnalysis.CSharp.EditorFeatures.dll and Microsoft.CodeAnalysis.VisualBasic.EditorFeatures.dll) has dependency to Microsoft.CodeAnalysis.EditorFeatures.Text package, so I don't think it should be added here